test/al_aws_collector_test.js

```diff
@@ -485,6 +511,22 @@ describe('al_aws_collector tests', function() {
                 });
             });
         });
+        
+        it('send log success with env vars not set', function (done) {
+            delete process.env.ingest_api;
+            delete process.env.azcollect_api;
+            AlAwsCollector.load().then(function (creds) {
+                var collector = new AlAwsCollector(
+                    context, 'paws', AlAwsCollector.IngestTypes.LOGMSGS, '1.0.0', creds);
+                var data = 'some-data';
+                collector.send(data, false, function (error) {
```

Comment:
same here. i would assert that endpoints was called and and correct vars are set.